external-builds/jax/README.md

```diff
@@ -0,0 +1,117 @@
+| ------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------- |
+| master        | ✅ Supported<br><ul><li>[ROCm/rocm-jax `master` branch](https://github.com/ROCm/rocm-jax/tree/master)</li></ul>                                                                                                                        | ❌ Not supported  |
+| 0.7.1         | ✅ Supported<br><ul><li>[ROCm/rocm-jax `rocm-jaxlib-v0.7.1` branch](https://github.com/ROCm/rocm-jax/tree/rocm-jaxlib-v0.7.1)</li></ul>                                                                                                | ❌ Not supported  |
+
+
+
+## Build instructions
+
+For the most up-to-date and detailed build steps, please refer to the [Quick Build instructions in the ROCm JAX repository](https://github.com/ROCm/rocm-jax/tree/master?tab=readme-ov-file#quickbuild). Follow those steps to ensure a successful build process.
+
+## Build jax_rocmX_plugin and jax_rocmX_pjrt wheels instructions
+
+To build `jax_rocmX_plugin` and `jax_rocmX_pjrt` wheels, please follow the official instructions provided in the ROCm JAX repository. The most accurate and up-to-date build steps are documented at [BUILDING.md#building](https://github.com/ROCm/rocm-jax/blob/master/BUILDING.md#building). Refer to this guide for environment setup, build commands, and troubleshooting tips.
+
+
+## Developer Setup
+
+For JAX development setup and the latest instructions, please refer to the [ROCm JAX DEVSETUP guide](https://github.com/ROCm/rocm-jax/blob/master/DEVSETUP.md). If you want to run JAX locally, follow the steps outlined in that document for environment preparation and build commands.
+
+## Running/testing JAX
+
+After building the wheels, you should verify their functionality by running the recommended tests. Please follow the instructions provided in the [ROCm JAX BUILDING.md - Running Tests section](https://github.com/ROCm/rocm-jax/blob/master/BUILDING.md#3-running-tests) to ensure your build passes all required checks. This guide covers environment setup, test commands, and troubleshooting tips for validating your wheel builds.
+
+## Nightly releases
+
+### Gating releases with JAX tests
+
+With passing builds, we upload `jaxlib`, `jax_pjrt`, and `jax_plugin` wheels to subfolders of the "v2-staging" directory in the nightly release S3 bucket with a public URL at https://rocm.nightlies.amd.com/v2-staging/
+
+Only after passing JAX tests do we promote validated wheels to the "v2" directory in the nightly release S3 bucket with a public URL at https://rocm.nightlies.amd.com/v2/
+
+If no runner is available: Promotion is blocked by default. Set `bypass_tests_for_releases=true` for exceptional cases under [`amdgpu_family_matrix.py`](/build_tools/github_actions/amdgpu_family_matrix.py)
```

Comment:
This does not look accurate for JAX yet. The code always uploads directly to v2 with no promotion process: https://github.com/ROCm/TheRock/blob/aae9d1c6ed01cd441c0337ac16a501b8581afb81/.github/workflows/build_linux_jax_wheels.yml#L96-L124

We may want code like what pytorch has: https://github.com/ROCm/TheRock/blob/aae9d1c6ed01cd441c0337ac16a501b8581afb81/.github/workflows/build_portable_linux_pytorch_wheels.yml#L201-L307